systemtests/util_test.go

```diff
@@ -28,7 +28,7 @@ var (
 
 func ClearEtcd(node remotessh.TestbedNode) {
 	logrus.Infof("Clearing etcd data")
-	node.RunCommand(`etcdctl rm --recursive /volplugin`)
+	node.RunCommand(`connwait 127.0.0.1:2379 && etcdctl rm --recursive /volplugin`)
```

Comment:
Nice... the more sleeps/waits we can replace with connwait, the better off we'll be!